### ISSUE TITLE
Function returns 400 error

### ISSUE THREAD

**Issue by faultoverload** [microsoftgraph/powershell-intune-samples#124]
https://github.com/microsoftgraph/powershell-intune-samples/blob/8b4f760a460839de6ee1726c3159a48478385edc/AppleEnrollment/AppleDEPProfile_Assign.ps1#L310

I have been unable to get this Function working with assigning a profile to a apple device. Has the required json payload possibly been changed?

`Invoke-RestMethod : {
  "error": {
    "code": "BadRequest",
    "message": "Resource not found for the segment 'depOnboardingSettings'.",
    "innerError": {
      "request-id": "06a907d7-c2c8-4b80-b423-2e5f278a1b74",
      "date": "2019-12-09T20:22:24"
    }
  }
}`

**Comment by davefalkus** [comment]
Hi @faultoverload 

I've just tested the script and it worked as expected, can you confirm you have APNS and a DEP token configured?

Thanks

Dave

**Comment by faultoverload** [comment]
@davefalkus I took another look at the script you wrote and compared it to the documentation and I believe I found out where I went wrong. The docs call for a 
`POST /deviceManagement/depOnboardingSettings/{depOnboardingSettingId}/enrollmentProfiles/{enrollmentProfileId}/updateDeviceProfileAssignment`

but your script actually generates a url like this.
`POST /deviceManagement/depOnboardingSettings/{depOnboardingSettingId}/enrollmentProfiles('{enrollmentProfileId}')/updateDeviceProfileAssignment`

When I use the properly formatted url my update requests work as expected. I believe I was trusting that the docs were right and I had a properly formatted url and it was somehow my json payload that was wrong.

Thanks for looking into this.
